src/test/java/bio/terra/service/dataset/DatasetDaoTest.java

```diff
@@ -904,6 +904,19 @@ public void createDatasetWithProperties() throws Exception {
     datasetDao.delete(datasetId, TEST_USER);
   }
 
+  @Test
+  public void updatePredictableFileIdsFlag() throws Exception {
+    UUID datasetId = createDataset("dataset-minimal.json");
+    assertFalse(
+        "predictable file ids flag is false",
+        datasetDao.retrieve(datasetId).hasPredictableFileIds());
+    datasetDao.setPredictableFileId(datasetId, true);
+    assertTrue(
+        "predictable file ids flag is false",
+        datasetDao.retrieve(datasetId).hasPredictableFileIds());
+    datasetDao.delete(datasetId, TEST_USER);
```

Comment:
Is there a reason that we delete at the end of the test rather than in an `@After` teardown method? (I see we do this in other tests in this class too)

Reply:
Yeah, this is an old test and I wanted to just follow convention (especially since some of the tests do a bunch of deleting as part of their tests.  I can update it though.  Lol, gonna my my line counts go up :)